code/modules/food_and_drinks/drinks/drinks.dm

```diff
@@ -299,6 +299,31 @@
 	desc = "An insult to Duke Purple is an insult to the Space Queen! Any proper gentleman will fight you, if you sully this tea."
 	list_reagents = list(/datum/reagent/consumable/tea = 30)
 
+/obj/item/reagent_containers/food/drinks/mug/tea/red
+	name = "Dutchess Red tea"
+	icon_state = "tea"
+	desc = "Dutchess Red personal blend of red tea leaves and hot water. Grate addtion for any meal."
+	list_reagents = list(/datum/reagent/consumable/tea/red = 30)
+
+/obj/item/reagent_containers/food/drinks/mug/tea/green
+	name = "Prince Green tea"
+	icon_state = "tea"
+	desc = "Prince Green's brew of tea. Well the blend may be different from time to time, Prince Green swares by it!"
```

Comment:
this should be
"Prince Green's brew of tea. The blend may be different from time to time, but Prince Green swears by it!"